server/server.cpp

```diff
@@ -23,15 +23,15 @@ std::vector<char> http_error_check(Http_Request* req, Metrics_Database* db) {
     Http_Response rep;
 
     int type = req->getType();
-    const char* metric = req->getMetric();
+    std::string metric(req->getMetric());
 
     if (type != Http_Request::GET){		
 	rep.Add_Header_Field("", "HTTP/1.1 400 Bad Request");
```

Comment:
Please configure the editor to not use tabs, but rather use spaces. Other people may use different sizes for the TAB (4 or 8 characters) which breaks alignment when viewing the code.